docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md

```diff
@@ -13,9 +13,46 @@ slug: avail-upgrade-validator-node
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-## Upgrade Process
+When upgrading Avail nodes there is two options for upgrading. Below two sections provide details on each upgrade process. You only need to use one of the below upgrade processes.
 
-Upgrading a Avail node involves a careful process to ensure a smooth transition without disruption to the network. Here's a step-by-step guide on how to upgrade a Avail node, 
+## Fast Upgrade
+
+While a rapid upgrade is possible, it carries inherent risks. For instance, if you proceed with the upgrade and encounter issues such as database corruption, there's a chance of prolonged downtime. This could lead to your node being removed from the active validator set due to unresponsiveness.
+
+The fast upgrade steps are:
+
+- Stop the Avail node.
+```
+sudo systemctl stop availd.service
+```
+
+- Locate your Avail binary, create a backup of the current binary, and then uninstall the existing binary by deleting the binary. Proceed to download the most recent binary announced in Discord, which will replace the previous version. 
```

Comment:
I think this should just link over to the installation page which will always have the official download links:

https://availproject.github.io/join-the-network/run-avail/full-node-setup

And say that the binary needs to be downloaded and placed in the same folder as where the old one was and started back up in the same way as before.

Reply:
updated to link to validator installation part ad most appropriate
https://availproject.github.io/join-the-network/run-avail/validator-node-setup

Left _announcement in discord_ part as the instruction has been to only install new binaries once testing is done and confirmed. For example testnet is running 1.6.2, there are two other binaries available after this version but all validators requested to stay on 1.6.2